### PR TITLE
[WC-864] Add clear and multiple options to Tags component.

### PR DESCRIPTION
**Pull request by myslaf**
Add clear and multiple options to Tags component.
Clear selected products, tags and users when we use clear button.
Add WebinarsParams type.
Add multiple tags filtering in webinars and courses.

https://escl24.atlassian.net/browse/WC-864

### CODE REVIEW

File: src/components/ProductsSelect/index.tsx [hunk 1]
```diff
@@ -67,7 +67,14 @@ export const ProductsSelect: React.FC<{
...
       style={{ width: '100%', minWidth: '150px' }}
       value={currProducts}
-      onChange={onChange}
+      onChange={(changedValue) => {
+        if (!changedValue) {
+          setCurrProducts([]);
+        }
+        if (onChange) {
+          onChange(changedValue);
+        }
```
Comment (qunabu) [inline src/components/ProductsSelect/index.tsx:76]: not very reactive - but i'm might not seeing whole context 

shouldn't be `useEffect` set to observable `CurrProducts` then fire `onChange` ?

File: src/components/Tags/index.tsx [hunk 2]
```diff
@@ -13,25 +13,54 @@ export const Tags: React.FC<{
...
+        if (typeof tag === 'object') {
+          return tag.title;
+        }
+        return tag;
+      });
+
+      setCurrTags(values);
+    }
+    return () => {
+      controller.abort();
```
Comment (qunabu) [inline src/components/Tags/index.tsx:43]: what are you aborting here ?

File: src/components/Tags/index.tsx [hunk 3]
```diff
@@ -13,25 +13,54 @@ export const Tags: React.FC<{
...
-      value={value}
+      mode={multiple ? 'multiple' : undefined}
+      onChange={(changedValue) => {
+        if (!changedValue) {
+          setCurrTags([]);
+        }
+        if (onChange) {
+          onChange(changedValue);
+        }
+      }}
```
Comment (qunabu) [inline src/components/Tags/index.tsx:62]: see comment above

**Review by qunabu — COMMENTED** [review]
please respond to comments

File: src/components/ProductsSelect/index.tsx (hunk 1 above)
Comment (myslaf) [reply src/components/ProductsSelect/index.tsx:76]: We need clear state where we handling selected products. onChange is optional prop. I can't do this in useEffect

File: src/components/Tags/index.tsx (hunk 2 above)
Comment (myslaf) [reply src/components/Tags/index.tsx:43]: I will remove this.

**Pull request merged**